src/css/_icons.scss

```diff
@@ -0,0 +1,29 @@
+// CSS styles for SVG icons used throughout video.js.
+//
+// The goal is to replace all icons from the font family pulled from videojs/font entirely.
+// This project currently uses fonts. We want to replace this with SVGs from
+// images/icons.svg. This will ensure consitency between versions, as well as simplified
+// and straight-forward customization.
+
+// Default styling for all SVG icons
+.vjs-svg-icon {
+  display: inline-block !important;
+  background-repeat: no-repeat;
+  background-position: center;
+
+  fill: #FFFFFF;
+  height: 16px;
+  width: 16px;
+
+  // Overwrite any font content
+  &:before {
+    content: none !important;
+  }
+}
```

Comment:
I'm trying to think of a better way.. without it, we basically have 2 icons overlapping each other because the font icons are added through css content. A different option would be to go through everywhere those fonts are added and do a `(:not)` check to ensure that the content is only being added when svgs are not supposed to be used, but not sure if that is a better solution.

We can remove this once we remove the font icons entirely, but this might be the best path for now.